Packs/Orca/Integrations/Orca/Orca.yml

```diff
@@ -1,117 +1,210 @@
+    required: false
+    type: 8
+    section: Collect
+    advanced: true
+  - additionalinfo: If this checked, all existing alerts will be fetched
+    display: Pull Existing Alerts
+    hidden: false
+    name: pull_existing_alerts
+    required: false
+    type: 8
+    section: Collect
+    advanced: true
 description: Agentless, Workload-Deep, Context-Aware Security and Compliance for AWS, Azure, and GCP.
 display: Orca
 name: Orca
 script:
   commands:
-  - arguments:
-    - default: false
-      description: Type of alert to get
-      isArray: false
-      name: alert_type
-      predefined:
-      - ''
-      required: false
-      secret: false
-    - default: false
-      description: Get alerts of a specific asset
-      isArray: false
-      name: asset_unique_id
-      required: false
-      secret: false
-    deprecated: false
-    description: Get the alerts on cloud assets
-    execution: false
-    name: orca-get-alerts
-    outputs:
-    - contextPath: Orca.Manager.Alerts
-      description: All alerts
-      type: String
-  - arguments:
-    - default: false
-      description: Asset unique id
-      isArray: false
-      name: asset_unique_id
-      required: true
-      secret: false
-    deprecated: false
-    description: Get Description of An asset
-    execution: false
-    name: orca-get-asset
-    outputs:
-    - contextPath: Orca.Manager.Asset
-      description: Asset description
-      type: String
-  dockerimage: demisto/python3:3.10.8.37233
+    - arguments:
+        - default: false
+          description: Type of alert to get
+          isArray: false
+          name: alert_type
+          predefined:
+            - ''
+          required: false
+          secret: false
+        - default: false
+          description: Get alerts of a specific asset
+          isArray: false
+          name: asset_unique_id
+          required: false
+          secret: false
+      deprecated: false
+      description: Get the alerts on cloud assets
+      execution: false
+      name: orca-get-alerts
+      outputs:
+        - contextPath: Orca.Manager.Alerts
+          description: All alerts
+          type: String
+    - arguments:
+        - default: false
+          description: Asset unique id
+          isArray: false
+          name: asset_unique_id
+          required: true
+          secret: false
+      deprecated: false
+      description: Get Description of An asset
+      execution: false
+      name: orca-get-asset
+      outputs:
+        - contextPath: Orca.Manager.Asset
+          description: Asset description
+          type: String
+    - name: orca-set-alert-severity
+      description: Change severity for the alert
+      arguments:
+        - name: alert_id
+          description: Alert id
+          required: true
+        - name: score
+          description: The Orca Score to set. Valid range 1.0 to 10.
+          required: true
+      outputs:
+        - contextPath: Orca.SetAlertSeverity.id
```

Comment:
same comment i mentioned in the code:
when updating an entity (like the alert in your case), no need to set an output key for that (SetAlertSeverity), since its not an entity itself, its just a change, so a human readable, a message it was changed and what is the change, should be enough, for the output field, you can use the Orca.Alert key (since its a key) .